src/kdalgorithms.h

```diff
@@ -52,6 +52,31 @@ void copy(InputContainer &&input, OutputContainer &output)
     std::copy(range.begin(), range.end(), detail::insert_wrapper(output));
 }
 
+// -------------------- copied --------------------
+template <typename ResultContainer, typename InputContainer>
+#if __cplusplus >= 202002L
+    requires ContainerOfType<ResultContainer, ValueType<InputContainer>>
+#endif
+ResultContainer copied(InputContainer &&input)
```

Comment:
Maybe add a `static_assert(not std::is_same...)` so that people don't start using it instead of copy constructors.

Reply:
This would mean removing line 68 and the test later on.
I agree with Ivan here, it's probably better to not use copied instead of the normal copy constructor, so a `static-assert` stating that would make sense IMO.